engine/flink/management/sample/src/main/resources/reference.conf

```diff
@@ -14,7 +14,10 @@ additionalPropertiesConfig {
   intOptionalProperty: {
     validators: [
       {
-        type: "LiteralIntValidator"
+        type: "RegExpParameterValidator"
```

Comment:
hmmm... this is sth we haven't thought of... :/ It's not so nice that here we have to write patterns in each occurence....

Reply:
We can rollback LiteralIntValidator.. ;)

Reply:
Indeed. Why it was changed? Now someone need to provide this boilerplate for handling common case (ints)? Configuration like this should be necessary only for custom patterns...

Reply:
Okey, I will roolback Integer verification ;)